packages/perspective-viewer-d3fc/src/js/tooltip/tooltip.js

```diff
@@ -34,7 +34,7 @@ function getTooltipDiv(container) {
 function generateHtml(tooltipDiv, data, settings) {
     // Add group data
     if (settings.crossValues.length) {
-        const groups = data.crossValue.split(",");
+        const groups = Array.isArray(data.crossValue) ? data.crossValue.split(",") : data.crossValue;
```

Comment:
This looks like it's the wrong way around?

Reply:
I think the issue in [IPJS-41](https://int.scottlogic.co.uk/jira/browse/IPJS-41) is that the label is a date rather than a string (not that it's an array).

The line is expecting a string and trying to turn it into an array, so something like this maybe?

````javascript
const groups = data.crossValue.split ? data.crossValue.split(",") : [data.crossValue];
````

Reply:
Right you are, of course!